test/trivial/vass/pragma-error-unrecognized-2.good

```diff
@@ -1 +1 @@
-pragma-error-unrecognized-2.chpl:2: error: unknown pragma: "this is a non-existing pragma"
+pragma-error-unrecognized-2.chpl:2: error: unknown pragma "this is a non-existing pragma"
```

Comment:
Did you intend to get rid of the colon?

Reply:
Yes. To me it reads more naturally without the colon, and I think it's more consistent with our use of colons in other errors -- we usually leave them out when simply naming something, which is what I see this as..